guides/LIBRARIES.md

```diff
@@ -0,0 +1,125 @@
+Library choices
+==================
+
+This is a partially-complete list of libraries used for different areas by Digital Publishing.
+
+The aims of this document are to define standards around what libraries are being used, provide a place to center conversation about changing libraries and be a reference point to the existance of internal libraries.
+
+## Table of Contents
+1. [Internal Library Standards](#Internal-Library-Standards)
+2. [HTTP and Middleware](#HTTP-and-middleware)
+3. [Config & Logging](#config-&-logging)
+4. [Database Drivers](#database-drivers)
+5. [Testing](#testing)
+6. [Misc.](#misc.)
+
+# Internal Library Standards
+
+We create internal libraries where we want our interaction with a 3rd-party library to be
+standardized across all of our services. Often this is as simple as adding our health 
+checking ability and surfacing the behaviour that's relevant to us, in other cases it 
+provides us the opportunity to set defaults consistently across our micro services.
+
+Our internal libraries typically begin with `dp-` in the repo name, and *must* reside in the
+`ONSdigital` Github organization. 
+  * Go note: It is acceptable for files to live at the top level of these repos and take the single word name after `dp-` as the package name. e.g. `dp-kafka` uses the `package kafka` definiton
+
+These client libraries all serve that high level wrapping purpose:
+  * [dp-api-clients-go](https://github.com/ONSdigital/dp-api-clients-go) - which houses all of our API clients
+  * [dp-elasticsearch](https://github.com/ONSdigital/dp-elasticsearch)
+  * [dp-nomad](https://github.com/ONSdigital/dp-nomad)
+  * [dp-vault](https://github.com/ONSdigital/dp-vault)
+  * [dp-mongodb](https://github.com/ONSdigital/dp-mongodb)
+  * [dp-s3](https://github.com/ONSdigital/dp-s3)
+  * [dp-kafka](https://github.com/ONSdigital/dp-kafka)
+
+The underlying drivers may be highlighted in further sections of this document, but where a library exists it should be used and updated rather than importing drivers directly to services.
+
+
+# HTTP and Middleware
+
+Internal libraries
+* [dp-net](https://github.com/ONSdigital/dp-net) wraps the Go `net/http` library with our preferred defaults
+* [dp-cookies](https://github.com/ONSdigital/dp-cookies)
+* [dp-healthcheck](https://github.com/ONSdigital/dp-healthcheck)
+* [dp-authorisation](https://github.com/ONSdigital/dp-authorisation)
+
+Internal forks of community libraries
+* ~~[restolino](https://github.com/ONSdigital/restolino) is our legacy Java REST framework~~ **DEPRECATED**
```

Comment:
If we're including `restolino` as a deprecated on, we should probably include `dp-cryptolite-java` too.

Reply:
`dp-ssqs` was a forked library from lloyd's repository